### PR TITLE
Mldb 1700 git checkout hash

### PR DESCRIPTION
**Pull request by mailletf**

### CODE REVIEW

**Comment by nicolaskruchten** [comment]
Is the pytsnic plugin test active? I honk it must be disabled because I
just fixed the plugin this morning...

On Thursday, 2 June 2016, François Maillet <notifications@github.com> wrote:

> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/mldbai/mldb/pull/406
> Commit Summary
>
>    - [MLDB-1700] When creating a plugin from a git repo, the commit hash
>    to checkout can now be specified
>    - Merge branch 'master' into MLDB-1700_git_checkout_hash
>
> File Changes
>
>    - *M* container_files/public_html/doc/builtin/lang/Python.md
>    <https://github.com/mldbai/mldb/pull/406/files#diff-0> (2)
>    - *M* server/plugin_resource.cc
>    <https://github.com/mldbai/mldb/pull/406/files#diff-1> (21)
>    - *M* testing/MLDB-878_experiment_proc.py
>    <https://github.com/mldbai/mldb/pull/406/files#diff-2> (3)
>    - *M* testing/mldb_python_plugin_test.cc
>    <https://github.com/mldbai/mldb/pull/406/files#diff-3> (26)
>    - *M* testing/pytanic_plugin_test.py
>    <https://github.com/mldbai/mldb/pull/406/files#diff-4> (12)
>
> Patch Links:
>
>    - https://github.com/mldbai/mldb/pull/406.patch
>    - https://github.com/mldbai/mldb/pull/406.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mldbai/mldb/pull/406>, or mute the thread
> <https://github.com/notifications/unsubscribe/AAMbA2-sMIidPJ_fNj3rhGhey49FKKMAks5qHv_KgaJpZM4IsuF7>
> .
>


-- 
Nicolas Kruchten, Eng  |  Staff Software Engineer at Datacratic  |
http://datacratic.com/

**Comment by mailletf** [comment]
appears disabled. and broken. checks for stuff that has been refactored out. so we need to update it

**Comment by nicolaskruchten** [comment]
doh!

File: testing/MLDB-878_experiment_proc.py [hunk 1]
```diff
@@ -77,8 +77,7 @@ def test_all(self):
         # did we run two training jobs that both got a good auc ?
         assert len(js_rez["status"]["folds"]) == 2
         for i in xrange(2):
-            assert js_rez["status"]["folds"][i]["resultsTest"]["auc"] > 0.95, \
-                'expected AUC %f to be above 0.95' % js_rez["status"]["folds"][i]["resultsTest"]["auc"]
+            self.assertGreater(js_rez["status"]["folds"][i]["resultsTest"]["auc"], 0.95)
```
Comment (nicolaskruchten) [inline testing/MLDB-878_experiment_proc.py]: this change is not strictly related but ok :)

**Comment by nicolaskruchten** [comment]
+1
